target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch

```diff
@@ -0,0 +1,5506 @@
+From 6f933347d0b4ed02d9534f5fa07f7b99f13eeaa1 Mon Sep 17 00:00:00 2001
```

Comment:
Something went wrong during the rebassing. This file should be deleted.

Reply:
Yes, I believe github threw two conflicts, that file and also `target/linux/lantiq/patches-5.4/0152-lantiq-VPE.patch`

I just deleted `target/linux/lantiq/patches-5.4/0001-MIPS-lantiq-add-pcie-driver.patch` .. shal I do the same for the other?

Reply:
You're right. Both files should be deleted.

Reply:
Done.